render/wlr_renderer.c

```diff
@@ -154,6 +154,15 @@ bool wlr_renderer_read_pixels(struct wlr_renderer *r, enum wl_shm_format fmt,
 		src_x, src_y, dst_x, dst_y, data);
 }
 
+bool wlr_renderer_blit_dmabuf(struct wlr_renderer *r,
```

Comment:
This function should assert that we're not in the middle of a rendering operation.